web/ts/README.md

```diff
@@ -0,0 +1,4 @@
+This directory contains JavaScript .js and TypeScript .ts files.  Modifications here will result in updates in the runtime ../js directory when rebuilt.
+
+Some files uses classes introduced in JavaScript [ES6](https://www.w3schools.com/js/js_es6.asp), so we use "es6" as the compilation target.
+
```

Comment:
@bobjacobsen 
One thing I forgot. It would be good to have a comment here on how to compile JavaScript and TypeScript files to get them to web/js. And if possible, which tools are needed to compile TypeScript.

Reply:
PR #10331 adds additional information about TypeScript.